specification/manufacturingdatasolution/ManufacturingDataSolution.Copilot/resources/instructionsApi.tsp

```diff
@@ -0,0 +1,77 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "./../models/instruction.tsp";
+
+using TypeSpec.Http;
+using Azure.Core;
+using Azure.Core.Traits;
+using TypeSpec.Versioning;
+using MdsSolution.CopilotService.Instructions.Models;
+
+@versioned(MdsSolution.CopilotService.Versions)
+namespace MdsSolution.CopilotService.Instructions;
+
+alias ServiceTraits = NoRepeatableRequests &
+  NoConditionalRequests &
+  NoClientRequestId;
+
+alias apiOperations = Azure.Core.ResourceOperations<ServiceTraits>;
+
+alias jobLocationResponseHeaders = ResponseHeadersTrait<
+  {
+    @doc("The background job location.")
+    @header
+    @pollingLocation
+    operationLocation: string;
+  },
+  TraitContext.Create | TraitContext.Update | TraitContext.Delete
+>;
+
+@doc("Instructions")
+interface Instructions {
+  @doc("Gets single instruction.")
+  get is apiOperations.ResourceRead<Instruction>;
+
+  @doc("List Instruction resources")
+  list is apiOperations.ResourceList<
+    Instruction,
+    QueryParametersTrait<{
+      @doc("The instruction status.")
+      @query
+      showActive: boolean;
+    }>
+  >;
+
+  @doc("Create an instruction.")
+  @pollingOperation(
+    MdsSolution.CopilotService.OperationStatus.OperationProgress.getOperationResult
+  )
+  create is apiOperations.LongRunningResourceCreateWithServiceProvidedName<
+    Instruction,
+    jobLocationResponseHeaders
+  >;
+
+  @doc("update instructions.")
+  @pollingOperation(
+    MdsSolution.CopilotService.OperationStatus.OperationProgress.getOperationResult
+  )
+  update is apiOperations.LongRunningResourceCreateOrUpdate<Instruction>;
```

Comment:
`LongRunningResourceCreateOrUpdate` is a PATCH

Guideline would not allow PATCH as LRO
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#lro-no-patch-lro

Would you see if this design is correct? Ask in API review meeting, if you need clarification.